From eed51b54253c303c593d9466ed8ed17523bda3d1 Mon Sep 17 00:00:00 2001
From: Stephan Kulow <coolo@suse.de>
Date: Wed, 15 Oct 2014 10:38:29 +0200
Subject: [PATCH 07/33] fixes for the opensuse template:

- add one more space for sources
- add empty lines in front of the warning preamble - otherwise format_spec_file
  removes it as old license comment
- don't %ghost %_bindir
---
 templates/gem_packages.spec.erb | 6 +++---
 templates/opensuse.spec.erb     | 4 +++-
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/templates/gem_packages.spec.erb b/templates/gem_packages.spec.erb
index d1734db..058660b 100644
--- a/templates/gem_packages.spec.erb
+++ b/templates/gem_packages.spec.erb
@@ -173,9 +173,9 @@ fi
 <% end -%>
 <% spec.executables.each do |executable| -%>
 <%= rb_bindir %>/<%= "#{executable}#{rb_suffix}-#{spec.version}" %>
-%ghost <%= rb_bindir %>/<%= "#{executable}#{rb_suffix}" %>
-%ghost <%= rb_bindir %>/<%= "#{executable}-#{spec.version}" %>
-%ghost <%= rb_bindir %>/<%= executable %>
+<%= rb_bindir %>/<%= "#{executable}#{rb_suffix}" %>
+<%= rb_bindir %>/<%= "#{executable}-#{spec.version}" %>
+<%= rb_bindir %>/<%= executable %>
 %ghost <%= rb_sysconfdir %>/alternatives/<%= executable %>
 %ghost <%= rb_sysconfdir %>/alternatives/<%= "#{executable}#{rb_suffix}" %>
 %ghost <%= rb_sysconfdir %>/alternatives/<%= "#{executable}-#{spec.version}" %>
diff --git a/templates/opensuse.spec.erb b/templates/opensuse.spec.erb
index 25fdec3..8bc281c 100644
--- a/templates/opensuse.spec.erb
+++ b/templates/opensuse.spec.erb
@@ -15,6 +15,8 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 <% if config && not(config.empty?) -%>
+
+
 #
 # This file was generated with a gem2rpm.yml and not just plain gem2rpm.
 # All sections marked as MANUAL, license headers, summaries and descriptions
@@ -74,7 +76,7 @@ Url:            <%= spec.homepage %>
 Source:         http://rubygems.org/gems/%{mod_full_name}.gem
 <% if config[:sources]
      config[:sources].each_with_index do |src, i| -%>
-Source<%= i+1 %>:       <%= src %>
+Source<%= i+1 %>:        <%= src %>
 <% end
  end -%>
 <% if config[:patches] -%>
-- 
2.26.2

